src/stko/_internal/calculators/openmm_calculators.py

```diff
@@ -12,6 +13,9 @@
 logger = logging.getLogger(__name__)
 
 
+warnings.filterwarnings("ignore", category=DeprecationWarning)
```

Comment:
why?

Reply:
Agreed, I got sick of seeing warnings, but that is not appropriate.